fs/operations/dedupe.go

```diff
@@ -209,6 +209,20 @@ func dedupeMergeDuplicateDirs(f fs.Fs, duplicateDirs [][]fs.Directory) error {
 	}
 	for _, dirs := range duplicateDirs {
 		if !fs.Config.DryRun {
+
+			//Find the largest directory and merge duplicates into it
```

Comment:
I think I'd factor this out into a function `findLargestDirectory(dirs []fs.Directory) (largestDirId int, err error)` then it will be easy to write a unit/integration test for it.